src/definitions.ts

```diff
@@ -87,17 +87,22 @@ export interface GoogleLoginOptions {
   grantOfflineAccess?: boolean;
 }
 
-export interface GoogleLoginResponse {
+export interface GoogleLoginOnlineResponse {
   accessToken: AccessToken | null;
-  idToken: string | null;
+  responseType: 'online';
```

Comment:
why idToekn is removed? It's use currently offline shouldn't disturb current usage